src/ontology/edits/oeo-social.omn

```diff
@@ -1693,6 +1701,19 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652",
         OEO_00040009
     
     
+Class: OEO_00020427
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A monetary price is a quality of a good that indicates the value in terms of the amount of money requested, expected, required or given in exchange for something else.",
```

Comment:
The definition part _... is a quality of a good ..._ should be axiomatised, e.g. as
`'monetary price' 'is quality of' some good`.

Reply:
Right. 
Since we are going to introduce `product` in #1912, I am wondering whether `monetary price` should refer to product, which comprises `good`?!

Reply:
Yes, `product` is even better as also service products can have monetary price.